### PR TITLE
Introduced new error code LedgerNotFound

### PR DESCRIPTION
**Pull request by dkulic**
Signed-off-by: Darko Kulic <darko.kulic@evernym.com>

### CODE REVIEW

File: libindy/src/api/mod.rs [hunk 1]
```diff
@@ -185,6 +185,9 @@ pub enum ErrorCode
     // Call pool.indy_set_protocol_version to set correct Protocol version.
     PoolIncompatibleProtocolVersion = 308,
 
+    // Item not found on ledger.
```
Comment (jovfer) [inline libindy/src/api/mod.rs]: @dkulic please add new error code to `libindy/includes/`

Comment (dkulic) [reply libindy/src/api/mod.rs]: Will do.

**Comment by jovfer** [comment]
@dkulic compilation is failed on jenkins. Please double-check by running `cargo test --no-run`

**Comment by dkulic** [comment]
bash-4.4$ cargo test --no-run
   Compiling indy v1.6.7 (file:///home/dkulic/work/indy-sdk/libindy)                                                                                                                                                                                                                                                         
    Finished dev [unoptimized + debuginfo] target(s) in 2m 45s

**Comment by jovfer** [comment]
@dkulic I've double-checked on my machine
Compilation of tests is failed both for 1.27.0 Rust and 1.29.1
Please make sure that you have no local uncommitted changes or outdated commits
`git log -1 && git status && cargo test --no-run`

**Comment by jovfer** [comment]
@dkulic at least this line can not be compiled https://github.com/hyperledger/indy-sdk/pull/1208/files#diff-5d71ae7e8973fe186466a758a72a8ba6R1920

File: libindy/tests/ledger.rs [hunk 2]
```diff
@@ -1891,6 +1891,35 @@ mod medium_cases {
...
+
+            utils::tear_down_with_wallet_and_pool(wallet_handle, pool_handle);
+        }
+
+        #[test]
+        #[cfg(feature = "local_nodes_pool")]
+        fn indy_get_parse_returns_error_for_wrong_type_and_unknown_schema() {
+            let (wallet_handle, pool_handle) = utils::setup_with_wallet_and_pool();
+
+git c            let get_schema_response = ledger::submit_request(pool_handle, &get_schema_request).unwrap();
```
Comment (jovfer) [inline libindy/tests/ledger.rs]: Compilation error

**Pull request merged**